akka-serialization-jackson/src/main/resources/reference.conf

```diff
@@ -24,9 +24,17 @@ akka.serialization.jackson {
 }
 #//#jackson-modules
 
+#//#algorithm
+akka.serialization.jackson {
+  # gzip - using common java gzip
+  # lz4 - LZ4 compression.
+  algorithm = gzip
+}
+#//#algorithm
+
 #//#compression
 akka.serialization.jackson {
-  # The serializer will compress the payload when it's larger than this value.
+  # For GZIP Only: The serializer will compress the payload when it's larger than this value.
   # Compression can be disabled with value 'off'.
```

Comment:
let's support `algorithm = off` to disable compression completely instead of having that on the `compress-larger-than`

Reply:
Added algorithm option 'off' and removed 'off' feature for `compress-larger-than`.